server/irmaserver/sessions.go

```diff
@@ -72,45 +80,86 @@ type memorySessionStore struct {
 	client    map[irma.ClientToken]*session
 }
 
+type redisSessionStore struct {
+	client *redis.Client
+	locker *redislock.Client
+	conf   *server.Configuration
+}
+
+type RedisError interface {
+	Error() string
+}
+
+type UnknownSessionError interface {
+	Error() string
+}
+
+const (
+	maxSessionLifetime         = 5 * time.Minute        // After this a session is cancelled
```

Comment:
fixed, reading the field now. I did modify it and use duration instead of an int in the config now (stricter typing + creating the duration only once).

Reply:
Making it a `time.Duration` has advantages, but it has created an inconsistency with the `schemes_update` and `max_request_age` options, which are conceptually also durations but are still stored as ints on the configuration structs. This is just the `irma server`; the keyshare server components probably also have time-based configuration.